RecipesAppTeamPlate/RecipesAppTeamPlate/VoidHandler.swift

```diff
@@ -0,0 +1,10 @@
+// VoidHandler.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+// swiftlint:disable all
+
+/// Создаем тип для пустой кложуры
+public typealias VoidHandler = () -> ()
```

Comment:
```suggestion
 typealias VoidHandler = () -> ()
```
создали но не использовали